src/shared/modules/discovery/discoveryDuck.ts

```diff
@@ -238,18 +238,20 @@ export const discoveryOnStartupEpic = (some$: any, store: any) => {
                 removeSearchParamsInBrowserHistory(
                   searchParamsToRemoveAfterAutoRedirect
                 )
-                const err = authRequestForSSO(SSORedirect)
-                if (err) {
-                  SSOError = err
+                try {
+                  authRequestForSSO(SSORedirect)
+                } catch (err) {
+                  SSOError = err.message
+                  authLog(err)
```

Comment:
shouldn't this log the error message vs the error object?

Reply:
You're right, updated